docs/contribute-module.md

```diff
@@ -13,12 +13,12 @@ Follow these steps to create or update a module.
 ## Before you begin
 
 - If you're running Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
-- Make sure that you have a current version of Python 3 and pip.
+- Make sure that you have a current version of Python 3 and pip ([Python installation instructions](python3-mac-installation.md)).
```

Comment:
I'd call out the mac instructions in a sub bullet as not all users are on mac

Reply:
Also highlight that the python version provided out of the box on mac is not suitable